<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мех: Супервоин</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="getStateButton">
       <property name="text">
        <string>Обновить состояние</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>606</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Ремонтных установок:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="repairKit">
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="text">
            <string>N/A</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="numOfCart">
           <property name="text">
            <string>Количество патронов:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="gunAmmoLabel">
           <property name="text">
            <string>N/A</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="repairButton">
         <property name="text">
          <string>Ремонт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="shootButton">
         <property name="text">
          <string>Выстрел</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Элемент:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="componentIdComboBox">
           <item>
            <property name="text">
             <string>Орудийный манипулятор</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Орудие</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Торс</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Хватательный манипулятор</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Левая нога</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Правая нога</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,7">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Дальность:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="shootComboBox">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>289</width>
      <height>200</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="4" column="0">
      <widget class="QLabel" name="leftLegLabel">
       <property name="text">
        <string>Левая нога</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QProgressBar" name="leftLegProgressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="gunLabel">
       <property name="text">
        <string>Орудие</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QProgressBar" name="gunManipProgressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="gunManipLabel">
       <property name="text">
        <string>Орудийный
манипулятор</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QProgressBar" name="gunProgressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QProgressBar" name="rightLegProgressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="roghtLegLabel">
       <property name="text">
        <string>Правая нога</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QProgressBar" name="grabManipProgressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="grabManipLabel">
       <property name="text">
        <string>Хвататательный
манипулятор</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="torsoProgressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="torsoLabel">
       <property name="text">
        <string>Торс</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>177</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="CurrentPositionLabel">
       <property name="text">
        <string>Текущее положение:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="xEdit">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yEdit">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>434</width>
      <height>154</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,10">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="moveLabel">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Движение</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item row="3" column="1">
          <widget class="QPushButton" name="moveBackwardButton">
           <property name="text">
            <string>↓</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="moveForwardButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>3</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>↑</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="moveLeftButton">
           <property name="text">
            <string>←</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="moveRightButton">
           <property name="text">
            <string>→</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,10">
       <item>
        <widget class="QLabel" name="turnLabel">
         <property name="text">
          <string>Поворот</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item row="2" column="1">
          <widget class="QPushButton" name="rotateRightButton">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="rotateLeftButton">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="mapArea">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>289</y>
      <width>951</width>
      <height>511</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>949</width>
       <height>509</height>
      </rect>
     </property>
     <widget class="MapWidget" name="mapWidget" native="true">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapWidget</class>
   <extends>QWidget</extends>
   <header>mapwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
